README.md

```diff
@@ -1,5 +1,6 @@
 # healthchecks
 
+[![Build Status](https://travis-ci.org/andela/healthchecks-f-society.svg)](https://travis-ci.org/andela/healthchecks-f-society)
```

Comment:
I did not expect you to copy it from the README and add it yourself...all you needed to do is run the rebase command against develop

i.e (while on your feature branch)
```shell
$ git rebase develop
```

This will fetch the changes on `develop` branch and prepend them to your changes.

Reply:
Undo the addition ☝️ then do so. This might have the effect of closing this PR, in which case you will have to recreate the PR from your feature branch.

This is about the discussion we were having about ensuring that you have the latest changes on the github repo.